docs/index.md

```diff
@@ -0,0 +1,21 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "ably Provider"
+subcategory: ""
+description: |-
+  
+---
+
+# ably Provider
+
+
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `token` (String, Sensitive)
+- `url` (String)
```

Comment:
Should URL be optional? Or maybe we can replace / add alternative parameter `version`?

Reply:
What would the version parameter be used for?

I'm thinking perhaps we can use a default url of "https://control.ably.net/v1". 
For testing, I'm using our staging endpoint with pre-prod credentials.

Reply:
I've made the url parameter optional now. In the provider, if url == "", we use a default of "https://control.ably.net/v1".